go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	go.opencensus.io v0.22.3
 	go.uber.org/zap v1.10.0
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
-	golang.org/x/sys v0.0.0-20200408040146-ea54a3c99b9b // indirect
+	golang.org/x/sys v0.0.0-20200408040146-ea54a3c99b9b
```

Comment:
Sneaking this change in. A direct reference to `x/sys` was actually introduced in https://github.com/open-telemetry/opentelemetry-collector/pull/953

Reply:
Would suggest to always split in separate PRs :)

Reply:
Alright 👍 Rolled this back in this branch